health/REFERENCE.md

```diff
@@ -129,6 +132,60 @@ You're interested in what comes after the comma: `disk.io`. That's the name of t
 
 If you create a template using the `disk.io` context, it will apply an alarm to every disk available on your system.
 
+#### Alarm line `class`
+
+Specify the classification of the alarm or template. 
+
+Class can be used to indicate the broader area of the system that the alarm applies to. For example, under the general `Database` class, you can group together alarms that operate on various database systems, like `MySQL`, `CockroachDB`, `CouchDB` etc. Example:
+
+```yaml
+class: Database
+```
+
+Netdata's stock alarms use the following `class` attributes by default, but feel free to adjust for your own requirements:
+
+| Class                    | Groups alarms for:                              |
+| ------------------------ | ----------------------------------------------- |
+| Ad Filtering             | Services related to Ad Filtering (like pi-hole) |
+| Certificates             | 
+| Cgroups                  |
+| Computing                | 
+| Containers               |
+| Database                 | Database systems (e.g. MySQL, Postgress, etc)   |
+| Data Sharing             |
+| DHCP                     |
+| DNS                      | 
+| Kubernetes               |
+| KV Storage               |
+| Linux                    | Services specific to Linux (e.g. systemd)       |
+| Messaging                |
+| Netdata                  | Internal Netdata components monitoring          |
+| Other                    |
+| Power Supply             |
+| Search engine            |
+| Storage                  |
+| System                   | General system alarms (e.g. cpu, network, etc.) |
+| Virtual Machine          | Virtual Machine software                        |
+| Web Proxy                |
+| Web Server               | Web server software (e.g. Apache, ngnix, etc.)  |
+| Windows                  |                                                 |
```

Comment:
A lot of classes have no _Groups alarms for_ descriptions. 

- Are we done with `class`ification? if not perhaps it makes sense to not add this table? @joelhans 

- And what's Other? do we use it?

- And i think we need to mentions that if not set we use `Unknown` for all the new attributes.

Reply:
Hi Ilya. Yes, we do have some `Other` as class in some alerts. I will add the `Unknown` case in the docs, thanks.

Reply:
I will add the rest of the descriptions, and go from there with Joel if needed. I suppose it makes sense to keep the table.

Reply:
Ok, just an idea - consider using _collapsible section_ feature for the tables to keep the doc tidy. Could be just me, feel free to ignore the suggestion if you don't like it.

<details>
<summary>default class attributes</summary>


| Class                    | Groups alarms for:                              |
| ------------------------ | ----------------------------------------------- |
| Ad Filtering             | Services related to Ad Filtering (like pi-hole) |
| Certificates             | 
| Cgroups                  |
| Computing                | 
| Containers               |
| Database                 | Database systems (e.g. MySQL, Postgress, etc)   |
| Data Sharing             |
| DHCP                     |
| DNS                      | 
| Kubernetes               |
| KV Storage               |
| Linux                    | Services specific to Linux (e.g. systemd)       |
| Messaging                |
| Netdata                  | Internal Netdata components monitoring          |
| Other                    |
| Power Supply             |
| Search engine            |
| Storage                  |
| System                   | General system alarms (e.g. cpu, network, etc.) |
| Virtual Machine          | Virtual Machine software                        |
| Web Proxy                |
| Web Server               | Web server software (e.g. Apache, ngnix, etc.)  |
| Windows                  |                                                 |

</details>

Reply:
Whoa! Neat! Added!